packages/hmr-plugin/src/actions/hmr-init.action.ts

```diff
@@ -0,0 +1,9 @@
+import { NgxsHmrSnapshot } from '../symbols';
+
+export class HmrInitAction {
+  static get type() {
+    return '@@HMR_INIT';
```

Comment:
Please add this disclaimer above the return statement:
```ts
// NOTE: Not necessary to declare the type in this way in your code. See https://github.com/ngxs/store/pull/644#issuecomment-436003138
```

Reply:
done